core/tests/test_utils.py

```diff
@@ -564,6 +598,28 @@ def tearDown(self):
     def _get_all_queue_names(self):
         return [q['name'] for q in self.taskqueue_stub.GetQueues()]
 
+    def enable_urlfetch_mock(self):
+        #pylint: disable=protected-access
+        if 'urlfetch' in apiproxy_stub_map.apiproxy._APIProxyStubMap__stub_map:
+            del apiproxy_stub_map.apiproxy\
+                    ._APIProxyStubMap__stub_map['urlfetch']
+        urlfetch_mock = URLFetchServiceMock()
+        apiproxy_stub_map.apiproxy.RegisterStub('urlfetch', urlfetch_mock)
+        self._urlfetch_mock = urlfetch_mock
+        #pylint: enable=protected-access
+
+    def disable_urlfetch_mock(self):
+        #pylint: disable=protected-access
+        if 'urlfetch' in apiproxy_stub_map.apiproxy._APIProxyStubMap__stub_map:
+            del apiproxy_stub_map.apiproxy\
+                    ._APIProxyStubMap__stub_map['urlfetch']
+        self._urlfetch_mock = None
+        self.testbed.init_urlfetch_stub()
+        #pylint: enable=protected-access
+
+    def set_urlfetch_return_values(self, **kwargs):
```

Comment:
I think it would be better if the args here were explicit (and their meaning documented).